docs/helpers.rst

```diff
@@ -170,6 +170,22 @@ To use `client` as an authenticated standard user, call its `login()` method bef
         response = client.get('/private')
         assert response.content == 'Protected Area'
 
+``async_client`` - ``django.test.AsyncClient``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An instance of a `django.test.AsyncClient`_
+
+.. _django.test.AsyncClient: https://docs.djangoproject.com/en/dev/topics/testing/tools/#the-test-client
+
+Example
+"""""""
+
+::
+
+    async def test_with_async_client(async_client):
```

Comment:
are we recommending people use:

```suggestion
    @asgiref.sync.async_to_sync
    async def test_with_async_client(async_client):
```
or

```suggestion
    @pytest.mark.asyncio
    async def test_with_async_client(async_client):
```

or something else?

Reply:
Yes! I forgot about it

Reply:
I wonder if `@asgiref.sync.async_to_sync` would actually work better

Reply:
> I wonder if `@asgiref.sync.async_to_sync` would actually work better

So, is you have some standard? Or comparison of it?

Reply:
It's listed in the Django docs

Reply:
Like, if you're using a single database connection and a tx for each new test then when Django supports async database backends and pytest-asyncio makes a new loop, the connection will be lost

Reply:
When Django's test runner encounters an `async def` test case, it automatically wraps it in `asgiref.async_to_sync()`. I don't know if it does anything special that pytest-asyncio doesn't. On the other hand it might be better not to mix the two. Really I am not an expert on this, I prefer gevent myself.